### PR TITLE
Add link to DSB Template for VS 2017, add documentation for VideoCaptureSample

### PR DESCRIPTION
**Pull request by zhuridartem**
Since new DSB template was already published, need to link to it from existing content.

### CODE REVIEW

File: _data/_samples.json [hunk 1]
```diff
@@ -692,6 +692,19 @@
       ]
     },
     {
+      "title": "Video Capture",
+      "description": "Create an app that initializes a web cam, takes pictures, and records video and audio.",
```
Comment (rachitb777) [inline _data/_samples.json]: Update this line.

File: en-US/Samples/VideoCapture.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
+---
+layout: sample
+title: Video Capture Sample
+description: Create an app that initializes a web cam, takes pictures, and records video and audio.
+keyword: Windows 10 IoT Core, wiring, arduino
```
Comment (rachitb777) [inline en-US/Samples/VideoCapture.md]: Update these as well.

File: _data/_samples.json (hunk 1 above)
Comment (zhuridartem) [reply _data/_samples.json]: Fixed

File: en-US/Samples/VideoCapture.md (hunk 2 above)
Comment (zhuridartem) [reply en-US/Samples/VideoCapture.md]: Fixed

File: en-US/Samples/VideoCapture.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+        await mediaCapture.StopPreviewAsync();
+        await mediaCapture.StopRecordAsync();
+        isRecording = false;
+    }
+}
+{% endhighlight %}
+
+### Play Captured Video
+
+After a file has been recorded, it can be played back by pressing `Play Captured VIdeo` button. This sets up a stream from the file to **MediaElement** XAML control.
```
Comment (rachitb777) [inline en-US/Samples/VideoCapture.md:174]: 'Play Captured Video'

**Pull request merged**